### PR TITLE
Misc metadata updates

### PR DESCRIPTION
**Pull request by jotaylor**
- Remove bad SZ102 visit D9 from STIS config.
- Add bad SZ102 visit D9 to rejected list, add bad V-BP-TAU exposure to rejected list
- Add quality comments for datasets with high CR rates, DQ=2048 removed, vignetting flagged, and stisblazefix applied
- Rename V-V1154-SCO-A back to V-V1154-SCO
- Update spectral type of cvso-104

### CODE REVIEW

**Comment by jotaylor** [comment]
Woops, I need to add the updated calibration DB, so marking this as a draft for now

File: README.md [hunk 1]
```diff
@@ -21,11 +21,20 @@ where `<version>` is the version of the ullyses-utils package you desire.
...
+This repo houses several utility scripts that are needed to creates ULLYSES HLSPs. Most are specific to this purpose, but some may be generally useful: specific functions in such scripts are described below.
+
+**`match_aliases.match_aliases`**
+
+Given an input target name, an attempt is made to match it to any aliases in the ULLYSES sample.
+
 ```
-from ullyses_utils import ullyses_config
+from match_aliases import match_aliases
+match_aliases("BAT99 105")
```
Comment (Rplesha) [inline README.md:33]: I think I would add here the different columns that are available to specify. I think I added this in the docstring so you might be able to copy/paste

**Review by Rplesha — CHANGES_REQUESTED** [review]
Small update to the readme, but otherwise looks good.

**Review by Rplesha — DISMISSED** [review]
Perfect!

**Comment by jotaylor** [comment]
@Rplesha I see there is a conflict for the Cal DB, unsurprisingly. I tried resolving it myself but there are many conflicting rows due to the number of quality comments I added. Do you think I should just scrap the updates to the DB here, merge the branch, then create a new branch with DB updates after I run the creation script?

**Comment by Rplesha** [comment]
@jotaylor I find it easiest to merge main into the current branch and run everything again if you want to give that a go. I think that should resolve the conflicts...potentially.

**Review by Rplesha — APPROVED** [review]

**Pull request merged**